### PR TITLE
lab-04-pair-programming

### PR DESCRIPTION
**Pull request by KAardal**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -14,71 +17,98 @@ for (var attempts = 4; attempts > 0 && notDone; attempts--){
       numCorrect++;
       notDone = false;
     } else if (guess > 5 ){
-        alert('Too high, guess again.');
```
Comment (Tenlia) [inline app.js]: good indentation improvement!

File: app.js [hunk 2]
```diff
@@ -14,71 +17,98 @@ for (var attempts = 4; attempts > 0 && notDone; attempts--){
...
-      } else if (guess < 5 ){
-        alert('Too low, guess again.');
+      alert('Too high, guess again.');
```
Comment (Tenlia) [inline app.js]: good indentation!

**Comment by Tenlia** [comment]
Excellent ACP-flow. It's always good to add and commit after you get something to work.

File: app.js [hunk 3]
```diff
@@ -14,71 +17,98 @@ for (var attempts = 4; attempts > 0 && notDone; attempts--){
...
-      notDone = false;
-    } else if (attempts === 1){
-      userResponse = 'You are out of guesses!';
+
+function questionTwo() {
+  var correctAnswer = ['Supra', 'Eclipse', 'M3', 'SLS AMG'];
+  var notDone = true;
+  var guess;
+  for (var attempts = 6; attempts > 0 && notDone; attempts--){
+    var userResponse = 'You are incorrect!'
```
Comment (morganelle) [inline app.js:36]: missing semicolon

File: app.js [hunk 4]
```diff
@@ -14,71 +17,98 @@ for (var attempts = 4; attempts > 0 && notDone; attempts--){
...
+    for (var i=0; i < correctAnswer.length && notDone; i++ ){
+      if(guess.trim().toLowerCase() === correctAnswer[i].trim().toLowerCase()){
+        userResponse = 'You are correct';
+        numCorrect++;
+        notDone = false;
+      } else if (attempts === 1){
+        userResponse = 'You are out of guesses!';
+      }
     }
+    alert(userResponse)
```
Comment (morganelle) [inline app.js:47]: missing semicolon

**Pull request merged**